### PR TITLE
API, histories: allow ordering of the index list using an 'order' que…

### PR DESCRIPTION
**Pull request by carlfeberhard**
…ry param

### CODE REVIEW

File: lib/galaxy/webapps/galaxy/api/histories.py [hunk 1]
```diff
@@ -90,6 +90,23 @@ def index( self, trans, deleted='False', **kwd ):
                     skip the first ( offset - 1 ) items and begin returning
                     at the Nth item
 
+        The list returned can be ordered using the optional parameter:
+            order:  string containing one the valid ordering attributes followed
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/histories.py]: s/one the/one of the/ ?

File: lib/galaxy/webapps/galaxy/api/histories.py [hunk 2]
```diff
@@ -117,7 +134,7 @@ def index( self, trans, deleted='False', **kwd ):
         filters += self.history_filters.parse_filters( filter_params )
 
         # TODO: eventually make order_by a param as well
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/histories.py]: The TODO can be removed, I guess.

**Comment by nsoranzo** [comment]
LGTM

**Comment by bgruening** [comment]
:+1:  from me too.

**Comment by jmchilton** [comment]
Looks great @carlfeberhard! It looks like the jobs API for instance has an order_by parameter that is a little different. We should probably followup and unify this convention across the API (at least in places where we expose this kind of option - may only be this and the jobs API though).

Update: Actually, it looks like your convention is a superset - so that is awesome.

**Pull request merged**